data/definitive-rubygems-authors-list.txt

```diff
@@ -0,0 +1,39 @@
+# This file has the definitive and declarative list of Samvera ruby
+# gems owners.  Each line must either be either an email address
+# of the associated Rubygems profile.
+aaron.collier@stanford.edu
+adam.wead@protonmail.com
+anna3lc@gmail.com
+armintor@gmail.com
+bess
+bmquinn
+cam156@psu.edu
+chris@cbeer.info
+cjcolvar
+dan.coughlin@gmail.com
+dchandekstark@gmail.com
+digger250@gmail.com
+drew@codhicitta.com
+dweber.consulting@gmail.com
+elrayle
+escowles@ticklefish.org
+glen.horton@gmail.com
+jenlindner@gmail.com
+jeremy.n.friesen@gmail.com
+jessie.keck@gmail.com
```

Comment:
hey @jkeck, do you want keys to the samvera castle or nah

Reply:
I find it useful to be able to contribute to some gems in samvera land that we depend on from time to time (although that seems to be fewer and father between).  I don't particularly mind, but if you all are trying to clean things up, feel free to 🔪 me.

Reply:
Nope, I think it's fine for you to retain access. Thought it prudent to ask first! Thanks, Jessie.